src/components/Select/SelectMenu/SelectMenuItem.tsx

```diff
@@ -17,6 +17,7 @@ export function SelectMenuItem(props: SelectMenuItemProps) {
   return (
     <li
       className={css(classes.item, selected && classes.selected, style)}
+      role='option'
```

Comment:
is this really correct? [Here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/option_role) it says that option MUST be contained by a listbox, is that?

Reply:
Great! this seems the best solution indeed.